.github/workflows/ci.yml

```diff
@@ -947,6 +948,11 @@ jobs:
             - suite: suite-delta-lake-databricks122
               ignore exclusion if: >-
                 ${{ env.CI_SKIP_SECRETS_PRESENCE_CHECKS != '' || secrets.DATABRICKS_TOKEN != '' }}
+            - suite: suite-delta-lake-http-metastore
+              config: hdp3
+            - suite: suite-delta-lake-http-metastore
+              ignore exclusion if: >-
+                ${{ env.CI_SKIP_SECRETS_PRESENCE_CHECKS != '' || secrets.DATABRICKS_TOKEN != '' }}
```

Comment:
Add the extra lines for the new secrets

https://github.com/trinodb/trino/blob/470f666be5a1cec41891963f842a7f3e6a808f1d/.github/workflows/ci.yml#L1048-L1054

https://github.com/trinodb/trino/blob/470f666be5a1cec41891963f842a7f3e6a808f1d/.github/workflows/ci.yml#L1115-L1121

Reply:
gentle reminder